apps/admin/frontend/src/screens/unconfigured_screen.tsx

```diff
@@ -359,47 +354,52 @@ export function UnconfiguredScreen(): JSX.Element {
   return (
     <NavigationScreen centerChild title="Configure VxAdmin">
       <Prose textCenter>
-        <p>How would you like to start?</p>
+        <P>How would you like to start?</P>
         {vxElectionFileIsInvalid && (
-          <Invalid>Invalid Vx Election Definition file.</Invalid>
+          <P>
+            <Font color="danger">
+              <Icons.Danger />
+            </Font>{' '}
+            Invalid Vx Election Definition file.
+          </P>
         )}
         {systemSettingsFileIsInvalid && (
-          <Invalid>Invalid System Settings file.</Invalid>
+          <P>
+            <Font color="danger">
+              <Icons.Danger />
+            </Font>{' '}
+            Invalid System Settings file.
+          </P>
         )}
-        <p>
+        <P>
           <FileInputButton
             accept=".json,application/json"
             onChange={handleVxElectionFile}
           >
             Select Existing Election Definition File
           </FileInputButton>
-        </p>
-        <HorizontalRule>or</HorizontalRule>
-        <p>
+        </P>
+        <P>
           <FileInputButton
             accept=".zip,application/zip"
             onChange={handleSetupPackageFile}
           >
             Select Existing Setup Package Zip File
           </FileInputButton>
-        </p>
-
-        {client && inputConversionFiles.length > 0 && (
-          <React.Fragment>
-            <HorizontalRule>or</HorizontalRule>
-            {!isUsingConverter && (
-              <Button onPress={() => setIsUsingConverter(true)}>
-                Convert from {client.getDisplayName()} files
-              </Button>
-            )}
-          </React.Fragment>
+        </P>
+
+        {client && inputConversionFiles.length > 0 && !isUsingConverter && (
+          <P>
+            <Button onPress={() => setIsUsingConverter(true)}>
+              Convert from {client.getDisplayName()} files
+            </Button>
+          </P>
         )}
-        <HorizontalRule>or</HorizontalRule>
-        <p>
+        <P>
           <Button onPress={loadDemoElection}>
             Load Demo Election Definition
           </Button>
-        </p>
+        </P>
       </Prose>
```

Comment:
@kofi-q is there a reason you're not removing `Prose` even though you marked it as deprecated?

Reply:
Ah, thanks for catching that - been meaning to remove those as I go

Reply:
✅  https://github.com/votingworks/vxsuite/pull/3597/commits/6982fdc1f88ceba8b43bf3fe5c5fb467478804cd